flank-scripts/src/main/kotlin/flank/scripts/release/updatebugsnag/UpdateBugSnagCommand.kt

```diff
@@ -0,0 +1,36 @@
+package flank.scripts.release.updatebugsnag
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.kittinunf.result.Result
+import flank.scripts.utils.ERROR_WHEN_RUNNING
+import flank.scripts.utils.SUCCESS
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withContext
+import kotlin.system.exitProcess
+
+class UpdateBugSnagCommand : CliktCommand(name = "updateBugsnag", help = "Update Bugnsag") {
+
+    private val bugsnagApiKey by option(help = "Bugsnag api key").required()
+    private val appVersion by option(help = "App version to update").required()
+    private val githubWorkflowUrl by option(help = "GitHub workflow url")
+
+    override fun run() {
+        runBlocking {
+            withContext(Dispatchers.IO) {
```

Comment:
I'm wondering if the changing dispatcher is necessary here.
Anyway, `runBlocking` can take `CoroutineContext` as the first argument, so `withContext` is redundant.

Reply:
removed